### ISSUE TITLE
[Request] Command module for Windows

### ISSUE THREAD

**Issue by absolutejam** [hubblestack/hubble-salt#74]
Howdy.

First off, I just want to say that I think this is a great project, and kudos on the work you've put into it!

Now, to extend this awesome project, I'd really like to see the command module ported to Windows. There are a lot of things I'd like to audit and I already have scripts that do so in PowerShell, but feeding them into this framework would be the finishing touch. This would let us Windows admins leverage the power of PowerShell & Pester, and all of Windows' other features that are available to raw commands (eg. WMI, COM objects).

Any thoughts?

**Comment by basepi** [comment]
This would be pretty straightforward. Since we're already using salt's `cmd.run` under the hood, all we would need to do is expose a couple more arguments to that function, like `shell` so you could pass in `powershell`, etc.

**Comment by absolutejam** [comment]
Trying to dig in to the code and have a play, but I can't seem to get my `command` entries in my profile to work - Any ideas?

Tried it on Linux first of all, and was going to then hack the existing `command` module on Windows.

```command:
  test1:
    data:
      '*':
        tag: test1
        commands:
          - 'echo 1':
               match: '1'
               match_output_regex: False
```

`/srv/pillar/top.sls`
```base:
  '*':
    - hubblestack:
        nova:
          enable_command_module: True```
```

**Comment by basepi** [comment]
I assume above your `test1` line you have a `command:` line?

Looks like your pillar data might not be quite right, it needs to be nested under a `hubblestack` key:

```
hubblestack:
  nova:
    enable_command_module: True
```

**Comment by absolutejam** [comment]
It was, but as you pointed out, I'd missed the `hubblestack:` key. I'll try and have a look at why the error wasn't being printed when I get a chance, amongst a few other things with this module :smile:

**Comment by absolutejam** [comment]
PR #81 submitted

**Comment by basepi** [comment]
Resolved by #81
